docs/fud/xilinx.md

```diff
@@ -46,6 +46,8 @@ The following commands enable remote usage of `vivado` and `vivado-hls` by defau
 
 The server must have `vivado` and `vivado_hls` available on the remote machine's path. (If you need the executable names to be something else, please file an issue.)
 
+To tell if this has been set up correctly, run `ssh username@havarti.cs.cornell.edu` (or the name of whichever server you are using to access Xilinx tools) and ensure that you are not prompted for a password. The `ssh-copy-id` command will let you setup your server to authenticate without a password.
+
```

Comment:
Might be worth splitting into general instructions and specific ones for havarti. The latter can go in a quote block. Also worth saying that after sshing, the vivado command should work without any source command needing to be run.

Fix and merge at your discretion!